packages/components/addon/components/hds/tabs/tab.hbs

```diff
@@ -8,7 +8,7 @@
   <a
     role="tab"
     class="hds-tabs__tab-link hds-typography-body-200 hds-font-weight-medium"
-    href={{concat "#" this.panelId}}
+    href={{if this.panelId (concat "#" this.panelId)}}
```

Comment:
it's a guard, to avoid the standalone `Tab` having an `href="#"` that when clicked would scroll the page to the top